app/src/main/java/com/khinthirisoe/bakingapp/ui/ingredients/StepsAdapter.kt

```diff
@@ -0,0 +1,62 @@
+package com.khinthirisoe.bakingapp.ui.ingredients
+
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.khinthirisoe.bakingapp.R
+import com.khinthirisoe.bakingapp.data.model.Step
+import com.khinthirisoe.bakingapp.ui.base.BaseViewHolder
+import com.khinthirisoe.bakingapp.ui.base.inflate
+import kotlinx.android.synthetic.main.list_steps.view.*
+
+class StepsAdapter(
+    private var steps: MutableList<Step>?,
+    private val clickListener: StepsAdapter.StepRecyclerViewClickListener) : RecyclerView.Adapter<StepsAdapter.ViewHolder>() {
+
+    var count = 0
```

Comment:
@KhinThiriSoe use position instead of count. When your list is long and when you scroll up, this count will be more than your list size